CorePayments/src/main/java/com/paypal/android/corepayments/common/DeviceInspector.kt

```diff
@@ -0,0 +1,21 @@
+package com.paypal.android.corepayments.common
+
+import android.content.Context
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class DeviceInspector(private val context: Context) {
+
+    fun isPayPalInstalled(): Boolean {
+        return isAppInstalled(PAYPAL_APP_PACKAGE)
+    }
```

Comment:
```suggestion
    val isPayPalInstalled: Boolean
      get() = isAppInstalled(PAYPAL_APP_PACKAGE)
```
nit: it seems like this could be a read-only property.